doc/python/marker-style.md

```diff
@@ -378,6 +378,31 @@ fig.show()
 
 ```
 
+#### Open Marker Colors
+
+In the previous example, each marker has two colors, a marker color (set in Plotly Express with `color="species"`)  and a line color (set on the line with `color="DarkSlateGrey"`. When using open markers, like `"diamond-open"` in the following example, the marker always uses only the marker color. Adding a line color won't update the marker. 
```

Comment:
Thanks @LiamConnors!  Is something like the following a clearer version of the second sentence?

All open markers, like `"diamond-open"` in the following example have a transparent fill which means user can specify only one color and that marker color parameter controls the line, and any dot or cross.  For open markers, the marker color is the only user controllable color and the line color does nothing.